Sources/SwiftKuerySQLite/SQLiteConnection.swift

```diff
@@ -459,3 +459,77 @@ public class SQLiteConnection: Connection {
         }
     }
 }
+
+class SQLiteColumnBuilder: ColumnCreator {
+    func buildColumn(for column: Column, using queryBuilder: QueryBuilder) -> String? {
+        guard let type = column.type else {
+            return nil
```

Comment:
Is it possible for error handling here?

I'm not sure under what circumstances column.type could be nil, but is it something we should alert a user to?

Reply:
The nil is handled in the calling function and an error thrown that identifies the column as having invalid attributes.